cmd/main.go

```diff
@@ -47,6 +47,9 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+	version  = "0.4.0-dev" // Change as versions are released
```

Comment:
This should be loaded from a version file.

Reply:
I'll seek to add that to the suggestion below and act on it.

Reply:
maybe something like this https://github.com/Kuadrant/limitador-operator/blob/main/version/version.go @ehearneRedHat